hugegraph-server/hugegraph-dist/download_keystore.sh

```diff
@@ -16,9 +16,20 @@
 # under the License.
 #
 
+curl --version >/dev/null 2>&1 ||
```

Comment:
remove a redundant docker-entrypoint.sh, not move the file to download_keystore.sh